client/packages/common/src/intl/locales/en/catalogue.json

```diff
@@ -31,5 +31,5 @@
   "title.months-of-stock": "Months of Stock",
   "title.packaging": "Packaging",
   "title.pricing": "Pricing",
-  "title.stock-on-hand": "Stock on Hand"
+  "title.stock-on-hand": "Available Stock"
```

Comment:
this is a sneaky offroad to address another issue: that the stock numbers don't align, between item details and stocktake. Reason being that one shows total stock and the other available stock. Hoping that this change clarifies a little.